pkg/util/log/logutils.go

```diff
@@ -13,16 +13,16 @@ import (
 )
 
 const (
-	projectFolder            = "/.gw/"
 	logrusStackJump          = 4
 	logrusFieldlessStackJump = 6
 )
 
-func SetLog(logLevel string, logfile bool, logFileName string) {
-	usr, _ := user.Current()
-	logFileFullPath := path.Join(usr.HomeDir, projectFolder, logFileName)
-	CreateProjectfolder(logFileFullPath)
-	if logfile {
+func SetLog(logLevel string, logFileName string) {
+	if logFileName != "" {
```

Comment:
is there a way to log to stdout and not a file?
It seems that if a path is not provided you use a default path under the home directory.

Reply:
If the file name id empty it will print to the stdout

Reply:
Where do you see that?
If no log file is set I think it should output to stderr.
This is what logrus defaults to, so in that case you don't need to call `logrus.SetOutput` at all.